source/extensions/filters/http/ext_proc/ext_proc.cc

```diff
@@ -14,6 +14,11 @@ namespace Envoy {
 namespace Extensions {
 namespace HttpFilters {
 namespace ExternalProcessing {
+namespace {
+
+const std::string ErrorPrefix = "ext_proc_error";
```

Comment:
👍  I replaced those with `constexpr absl::string_view` but had to add one ABSL_ATTRIBUTE_UNUSED for FilterName because it is not used anywhere. With std::string without constexpr it had generated init code that was referencing the global variable but with constexpr absl::string_view it's now clearly unreferenced and need unused. I don't think we also want to get rid of this var.